README.md

```diff
@@ -1,76 +1,43 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome books list using Java Script!
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+In this project, we built a simple website that allows users to add and remove books from a list using JavaScript objects and arrays. We dynamically modify the DOM and add basic events.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+- Java Script
+- HTML
+- CSS
 
-To get a local copy up and running follow these simple example steps.
+## Deployment Link
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+https://timowest12.github.io/Awesome-books/
```

Comment:
- [ ] OPTIONAL
It will be a nice idea to keep your demo link's _URL_ implicit, like doing this in your markdown:
`[Live Demo](https://timowest12.github.io/Awesome-books/)`